.changelog/3733.breaking.txt

```diff
@@ -0,0 +1,32 @@
+chore(instrumentation): move instrumentation configuration from opentelemetry-operator values
+
+#### Moved:
+* From `opentelemetry-operator.instrumentationJobImage` to `instrumentation.instrumentationJobImage`
+* From `opentelemetry-operator.createDefaultInstrumentation` to `instrumentation.createDefaultInstrumentation`
+* From `opentelemetry-operator.instrumentationNamespaces` to `instrumentation.instrumentationNamespaces`
+* From `opentelemetry-operator.instrumentation.dotnet.traces` to `instrumentation.dotnet.traces`
+* From `opentelemetry-operator.instrumentation.dotnet.metrics` to `instrumentation.dotnet.metrics`
+* From `opentelemetry-operator.instrumentation.dotnet.extraEnvVars` to `instrumentation.dotnet.extraEnvVars`
+* From `opentelemetry-operator.instrumentation.java.traces` to `instrumentation.java.traces`
+* From `opentelemetry-operator.instrumentation.java.metrics` to `instrumentation.java.metrics`
+* From `opentelemetry-operator.instrumentation.java.extraEnvVars` to `instrumentation.java.extraEnvVars`
+* From `opentelemetry-operator.instrumentation.nodejs` to `instrumentation.nodejs`
+* From `opentelemetry-operator.instrumentation.python.traces` to `instrumentation.python.traces`
+* From `opentelemetry-operator.instrumentation.python.metrics` to `instrumentation.python.metrics`
+* From `opentelemetry-operator.instrumentation.python.extraEnvVars` to `instrumentation.python.extraEnvVars`
+
+#### Changed:
```

Comment:
What is the difference between `moved` and `changed`?

Reply:
Moved are the keys `moved` from `opentelemetry-operator` key to `instrumentation`. Changed are keys which stay in the `opentelemetry-operator` but in different place (if it makes sense :))

Reply:
It makes sense, but not sure if the distinguishment is significant and needed for customer